daml-assistant/daml-helper/src/DamlHelper/Main.hs

```diff
@@ -35,7 +35,7 @@ data Command
     | New { targetFolder :: FilePath, templateNameM :: Maybe String }
     | Migrate { targetFolder :: FilePath, mainPath :: FilePath, pkgPathFrom :: FilePath, pkgPathTo :: FilePath }
     | Init { targetFolderM :: Maybe FilePath }
-    | Deploy { optSandboxPort :: Maybe SandboxPort }
+    | Deploy { optHostname :: Maybe String, optSandboxPort :: Maybe SandboxPort }
```

Comment:
Is this the host where the sandbox is running? If so, why not `optSandboxHost` for the sake of consistency?

Reply:
We probably want to move the other direction, and talks about Ledgers, not Sandbox.
This was some feedback we got from users of the `deploy` command.